advanced/editor-control-identifiers.md

```diff
@@ -38,11 +38,21 @@ Use the [toolbar]({{ site.baseurl }}/configure/editor-appearance/#toolbar) or th
 | removeformat | core | Removes the formatting from the current selection. |
 | subscript | core | Applies subscript format to the current selection. |
 | superscript | core | Applies superscript format to the current selection. |
+| forecolor | core | Applies foreground/text color to selection. |
+| backcolor | core | Applies background color to selection. |
 | visualaid | core | Toggles the visual aids for invisible elements. |
-| insert | core | Groups various insert actions into one button. |
+| h1 | core | Changes current line to the "Heading 1" style. |
+| h2 | core | Changes current line to the "Heading 2" style. |
+| h3 | core | Changes current line to the "Heading 3" style. |
+| h4 | core | Changes current line to the "Heading 4" style. |
+| h5 | core | Changes current line to the "Heading 5" style. |
+| h6 | core | Changes current line to the "Heading 6" style. |
+| remove | core | Removes (deletes) the selected content or the content before the cursor position. |
+| selectall | core | Selects all content in the editor. |
 | hr | [hr]({{ site.baseurl }}/plugins/hr/) | Inserts a horizontal rule into the editor. |
```

Comment:
Atm this list seems in a rather random order, just grouped by plugin. Maybe we should reorder it so the plugins are in alphabetical order?

Reply:
yeah, I wasn't sure what would be the least confusing way to order these the table...

Reply:
Yeah I wasn't sure either, but I figured some order was better than no order at all :man_shrugging: